treeherder/seta/runnable_jobs.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import requests
+from redo import retry
+
+HEADERS = {
+    'Accept': 'application/json',
+    'User-Agent': 'treeherder-seta',
+}
+LOG = logging.getLogger(__name__)
```

Comment:
So in these sorts of cases, it's IMO best to use the conventions elsewhere in the project, even if you have your own convention that you prefer. Other people (especially new contributors) get thrown off when we're not consistent.